clients/Application/src/app/views/error/unauthorized.component.ts

```diff
@@ -55,31 +63,35 @@ export class UnauthorizedComponent implements OnInit {
   }
 
   hasRequiredError(field: string) {
-    return this.hasError(field, 'required');
+    return !!this.tenantForm.get(field)?.hasError('required');
   }
 
-  hasError(field: string, error: any) {
-    const formField = this.tenantForm.get(field);
-    return !!formField.errors[error];
+  openErrorMessageSnackBar(errorMessage: string) {
+    this._snackBar.open(errorMessage, 'Dismiss', {
+      duration: 4 * 1000, // seconds
+    });
   }
 
-
   login() {
-
     let tenantName = this.tenantForm.value.tenantName;
-    this.authConfigService.setTenantConfig(tenantName)
-     .then((val)=>{
-      this.oidcSecurityService.authorize();
-
-    }).catch((errorResponse) => {
+    if (!tenantName) {
+      this.errorMessage = 'No tenant name provided.';
       this.error = true;
-      this.errorMessage = errorResponse.error.message;
-
-    });
+      this.openErrorMessageSnackBar(this.errorMessage);
+      return false;
+    }
+
+    this.authConfigService
+      .setTenantConfig(tenantName)
+      .then((val) => {
+        this.router.navigate(['/dashboard']);
```

Comment:
How is it routing to login? by routing to '/dashboard'

Reply:
`<amplify-authenticator>` components takes care of checking for a session and routing the user to the login page when needed.